app/containers/ActionView/index.js

```diff
@@ -148,9 +173,11 @@ const mapStateToProps = (state, props) => ({
         path: 'categories',
         key: 'taxonomy_id',
         reverse: true,
-        join: {
```

Comment:
@tony-luisi the join here is correct as it imposes a condition on the returned categories (we only want the ones that are connected). Extend on the other hand will add additional table entries (here from the measure_categories table) to each category (assigned or not assigned). This is useful for editing when we want both the assigned and un-assigned categories for users to chose from.

Maybe we should find a better name for "join", eg "joinWhere" to strengthen its conditional purpose